README.md

```diff
@@ -166,7 +166,7 @@ settings:
     > repository that you want to set up (you may need to "Resync") and hit the
     > "Configure" button. The token should show up in the "Coverage" tab.
 
-- `AUTO_UPDATE_GITHUB_TOKEN`: GitHub token with permissions to write to the
+- `GITHUB_TOKEN`: GitHub token with permissions to write to the
     repository.
```

Comment:
Please mention that this is auto-generated. We don't want anyone to set this manually and risk overriding the auto-generated one.

Also, I would shorten the text from the note (it's not important when the need for granting the authority was introduced) and include it in the variable description, e.g., sth like this:

> `GITHUB_TOKEN`: This token is auto-generated by GitHub Actions for each job, so please do **NOT** set it manually. However, the token needs to be granted the authority to create pull requests. See [GitHub][gh-update-token] for details on how to set this permission.